evalai/utils/challenges.py

```diff
@@ -114,15 +148,32 @@ def display_ongoing_challenge_list():
 
 def display_future_challenge_list():
     """
-    Displays the list of future challenges from the backend
+    Fetch and display upcoming challenges
     """
     url = "{}{}".format(get_host_url(), URLS.future_challenge_list.value)
     display_challenges(url)
 
 
 def get_participant_or_host_teams(url):
     """
-    Returns the participant or host teams corresponding to the user
+    Fetch the participant or host teams corresponding to the URL
+
+    Args
+    ----------
+    url: string
+        Team url according to the flag passed
+
+    Raises
+    -------
+    requests.exceptions.HTTPError
+        Server throws 4XX error
+    requests.exceptions.RequestException
+        Server throws request exception
+
+    Returns
+    -------
+    Results: List
+       List of participant or host teams
```

Comment:
Please maintain one order for writing doc-strings. It should be `Args`, `Returns`, `Raises`.

Reply:
@RishabhJain2018 What if the function returns nothing? Should I not mention that label `Returns` or leave it blank?

Reply:
Add the `Returns` label and below it write None. E.g.

```
Returns
-------
None
```